test/domain.test.ts

```diff
@@ -485,6 +486,81 @@ describe('store: [adapter=DOMAIN]', () => {
     });
   });
 
+  describe('Reset():', () => {
+    it('should reset the store by removing all values', (done) => {
+      const callback = () => {
+        globalStore.set({ foo: 'foo', bar: 'bar' });
+
+        expect(globalStore.isInitialized()).to.equal(true);
+
+        expect(globalStore.get('foo')).to.equal('foo');
+        expect(globalStore.get('bar')).to.equal('bar');
+
+        done();
+      };
+
+      globalStore.initialize(adapter)(callback);
+
+      expect(globalStore.reset).to.not.throw('Async store not initialized.');
+
+      globalStore.reset();
+
+      expect(globalStore.isInitialized()).to.equal(false);
+
+      const activeDomain = getActiveDomain();
+      expect(activeDomain[STORE_KEY]).to.equal(null);
+    });
+
+    it('should do nothing if the store is not initialized.', () => {
+      expect(globalStore.reset).to.throw('Async store not initialized.');
+    });
+
+
+  });
+
+  describe('del():', () => {
+    it('should delete a key from the store.', (done) => {
+      const callback = () => {
+        globalStore.set({ foo: 'foo', bar: 'bar' });
+
+        expect(globalStore.get('foo')).to.equal('foo');
+        expect(globalStore.get('bar')).to.equal('bar');
+
+        globalStore.del('foo');
+
+        expect(globalStore.get('foo')).to.equal(undefined);
+        expect(globalStore.get('bar')).to.equal('bar');
+
+        done();
+      };
+
+      globalStore.initialize(adapter)(callback);
+    });
+
+    it('should do nothing if the key does not exist.', (done) => {
+      const callback = () => {
+        globalStore.set({ foo: 'foo', bar: 'bar' });
+
+        expect(globalStore.get('foo')).to.equal('foo');
+        expect(globalStore.get('bar')).to.equal('bar');
+
+        expect(globalStore.del.bind(globalStore, 'baz')).to.not.throw('Async store not initialized.');
```

Comment:
We don't need this expect here as you have a separate test if the store is not initialized.